test/e2e/specs/site-editor/style-book.spec.js

```diff
@@ -59,37 +59,45 @@ test.describe( 'Style Book', () => {
 		).toBeVisible();
 		await expect( page.locator( 'role=tab[name="Theme"i]' ) ).toBeVisible();
 
+		// Buttons to select block examples are rendered within the Style Book iframe.
+		const styleBookIframe = page.frameLocator(
+			'[name="style-book-canvas"]'
+		);
+
 		await expect(
-			page.locator(
-				'role=button[name="Open Headings styles in Styles panel"i]'
-			)
+			styleBookIframe.getByRole( 'button', {
+				name: 'Open Headings styles in Styles panel',
+			} )
 		).toBeVisible();
 		await expect(
-			page.locator(
-				'role=button[name="Open Paragraph styles in Styles panel"i]'
-			)
+			styleBookIframe.getByRole( 'button', {
+				name: 'Open Paragraph styles in Styles panel',
+			} )
 		).toBeVisible();
 
 		await page.click( 'role=tab[name="Media"i]' );
 
 		await expect(
-			page.locator(
-				'role=button[name="Open Image styles in Styles panel"i]'
-			)
+			styleBookIframe.getByRole( 'button', {
+				name: 'Open Image styles in Styles panel',
+			} )
 		).toBeVisible();
 		await expect(
-			page.locator(
-				'role=button[name="Open Gallery styles in Styles panel"i]'
-			)
+			styleBookIframe.getByRole( 'button', {
+				name: 'Open Gallery styles in Styles panel',
+			} )
 		).toBeVisible();
 	} );
 
 	test( 'should open correct Global Styles panel when example is clicked', async ( {
 		page,
 	} ) => {
-		await page.click(
-			'role=button[name="Open Headings styles in Styles panel"i]'
-		);
+		await page
+			.frameLocator( '.edit-site-style-book__iframe' )
```

Comment:
Missed this? 😆

Reply:
I was too excited to commit, I totally missed this one, thanks for the eagle eyes! 👀 😀

Reply:
There's one more below too 😆

Reply:
Hang on, isn't that one already updated?

Reply:
Oh, oops! Right! GitHub failed me somehow 😅 .

Reply:
Thanks for confirming — I've been staring at this diff for too long, so I thought I very well could have missed something 😄